examples/example_parameters.param

```diff
@@ -0,0 +1,106 @@
+lambdaFactor 1
+
+# Levels
+# ------
+# Levels specify the resolution of the input images and the parameters of the multilevel minimization. The multilevel algorithm expects the input images to be quadratic and to have a dyadic resolution, i.e. the number of pixels in one coordinate direction is a power of two. The exponent is called level. For instance, level 8 means that the input images have $2^8=256$ pixels in each direction. Thus, 9 and 10 encode 512 and 1024 pixels respectively.
+# Most of these should be filled out by default if you use the python wrapper.
+# The precision of the input images = log_2(image sidelength) and the size of the output
+precisionLevel 8
+# The stopping level of the grid. i.e. deformations are not calculated beyond this resolution even if images have more pixels. If you want the resolution to equal the resolution of the images, stoplevel = precisionLevel. Higher levels take longer to compute.
+stopLevel 8
+# the starting level of the grid, i.e. the optimization will first be done on a grid of size 2^startLevel x 2^startlevel. startLevel should be small enough so that you still see the individual atoms if the input images are downsampled to this size. Usually startLevel is one or two smaller than stopLevel.
+startLevel 6
+# In the refinement step start at this level, usually >startLevel and <=stopLevel.
+refineStartLevel 7
+# In the refinement step stop at this level
+refineStopLevel 8
+# ?
+checkboxWidth 8
+# Resize the input images to the stop level, if different from precision level.
+resizeInput 0
+
+# In terms of Algorithm 1 of the Ultramicroscopy paper, we have $m_0=$ `startLevel` and $m_1=$ `stopLevel` in the first for loop of Algorithm 2, and $m_0=$ `refineStartLevel` and $m_1=$ `refineStopLevel` for the second for loop in Algorithm 2.
+
+# Miscellaneous
+# -------------
+# Number of times the entire calculation is performed
+numExtraStages 2
+# Adjustment of lambda in subsequent stages. Usually <1 because needed adjustments are minor
+extraStagesLambdaFactor 0.1
+# Use bilinear interpolation versus nearest neighbors when going to next step (level)
+resampleInsteadOfProlongateDeformation 1
+# If 0 compare images sequentially, if 1 compare images always to the first frame
+dontAccumulateDeformation 0
+# Compare to the median image if true, else compare to the mean
+useMedianAsNewTarget 1
+# If you already calculated stage 1, skip it if True
+skipStage1 0
+# If you use the previous option, use the results from that step or not
+reuseStage1Results 1
+# Remove the bias from comparing to the first frame.
+reduceDeformations 1
+
+##############
+### Output ###
+##############
+# Folder where the output is saved
+saveDirectory " "
+# Only save the displacements
+calcInverseDeformation 0
+# Options below mainly options for debugging and saving out more output beyond displacements
```

Comment:
The comments for `calcInverseDeformation` and `onlySaveDisplacement` are switched.